app/controllers/hooks.js

```diff
@@ -0,0 +1,39 @@
+var variablesHelper = require("../helpers/variables");
+var eventListener = require("./listener");
+
+var onConfigurationChanged = function onConfigurationChanged() {
+	console.log("Updating Event handler listener config (configuration changed)");
```

Comment:
Moet hier geen eslint ignore worden toegevoegd? (ook bij de andere logs in dit bestand)